### ISSUE TITLE
SAML integration for SSO and keeping session open

### ISSUE THREAD

**Issue by pafcioooo** [joshdrummond/webpasswordsafe#114]
If someone uses webpasswordsafe extensively then it is very annoying to enter passwors when session expires - even if it is set to 1 hours.Esspecily when your user name is lik name.surname@domain and password isnot trivial. 
To solve this in web environment we can add SAML/Oauth support so one can login with any SAML or OAuth provider eg. Windows ADFS or Google or even FB (of course if there is account already set).

**Comment by zeroluck** [comment]
I agree with this. I think that if OAuth providers support is too difficult to implement or it doesn't happen to get done, at the very least we should revise how the login and timeout relogin process works. I've submitted an issue for this here: https://github.com/joshdrummond/webpasswordsafe/issues/117

**Comment by katgirl** [comment]
see also #60
